src/jvmMain/kotlin/io/github/fourlastor/wilds_launcher/states/UpdateCheckerState.kt

```diff
@@ -0,0 +1,3 @@
+package io.github.fourlastor.wilds_launcher.states
+
+class UpdateCheckerState : State
```

Comment:
If your class does not have parameters, it can be an `object` (effectively a singleton)

```suggestion
object UpdateCheckerState : State
```